driver/terraform.go

```diff
@@ -170,6 +170,14 @@ func (tf *Terraform) InitTask(ctx context.Context) error {
 	return tf.initTask(ctx)
 }
 
+// DestroyTask deletes the task on the driver
+func (tf *Terraform) DestroyTask(ctx context.Context) {
```

Comment:
nit: suggestion to update this comment to move away from "delete" language

Reply:
> makes sense to me. thanks for taking the time to chat a bunch about naming!

No problem! glad we got it figured out :D

Reply:
thanks for catching